rfcs/top-navigation-silhouette.md

```diff
@@ -0,0 +1,67 @@
+# Top Navigation Silhouette
+
+This silhouette provides proper styles for wrapper element to be used as Top Navigation. By default element has no visual styles.
+Silhouettes comply with Base UI grid layout, so they are responsive to resizing and stretching, changing sizes, fonts, etc
+
+## Usage
+
+### Basic usage
+
+```javascript
+import * as React from 'react';
+import {TopNavigation} from 'baseui/silhouette/top-navigation';
```

Comment:
Can we have it imported from `baseui/silhouette` with no subdirs?

Reply:
yes. last proposal is to have import links from silhouette folder directly